src/org/usfirst/frc/team5590/robot/commands/CloseGrabber.java

```diff
@@ -0,0 +1,42 @@
+package org.usfirst.frc.team5590.robot.commands;
+
+import org.usfirst.frc.team5590.robot.Robot;
+
+import edu.wpi.first.wpilibj.command.Command;
+
+/**
+ *
+ */
+public class CloseGrabber extends Command {
+
+	public CloseGrabber() {
+		// Use requires() here to declare subsystem dependencies
+		// eg. requires(chassis);
+		requires(Robot.grabber);
+	}
+
+	// Called just before this Command runs the first time
+	protected void initialize() {
+		System.out.println("Starting CloseGrabber");
+	}
+
+	// Called repeatedly when this Command is scheduled to run
+	protected void execute() {
+		Robot.grabber.clawsClose();
+	}
+
+	// Make this return true when this Command no longer needs to run execute()
+	protected boolean isFinished() {
+		return true;
+	}
+
+	// Called once after isFinished returns true
+	protected void end() {
+		System.out.println("Ending CloseGrabber");
+	}
+
+	// Called when another command which requires one or more of the same
+	// subsystems is scheduled to run
+	protected void interrupted() {
+	}
+}
```

Comment:
Small thing, but add an empty new line at the end of the file. It makes GitHub and java happier.